src/core/operation.h

```diff
@@ -397,6 +401,7 @@ QuicOperationEnqueueFront(
 _IRQL_requires_max_(DISPATCH_LEVEL)
 QUIC_OPERATION*
 QuicOperationDequeue(
+    _In_ QUIC_PARTITION* Partition,
     _In_ QUIC_OPERATION_QUEUE* OperQ
```

Comment:
nit: All functions above take the queue as the first parameter, but the last two take the partition first.
It'd be nice to be consistent.

Reply:
I actually wonder if given the number of helpers that need the partition, adding the pointer as an OperQ member would make sense.